includes/admin/class-wc-admin-dashboard.php

```diff
@@ -345,7 +345,7 @@ public function network_orders() {
 			<div class="post-type-shop_order">
 			<div id="woocommerce-network-order-table-loading" class="woocommerce-network-order-table-loading is-active">
 				<p>
-					<span class="spinner is-active"></span> <?php esc_html_e( 'Loading network orders', 'woocommerce' ); ?>
+					<span class="spinner is-active"></span><?php esc_html_e( 'Loading network orders', 'woocommerce' ); ?>
```

Comment:
Could you please clarify why did you opt to remove this space? I didn't test to see if it has any consequences (and I assume it is unlikely to have) but asking anyway as this change doesn't seem related to this PR and it is not mentioned in the PR description, so maybe it was added by mistake?

Reply:
Yes, it was removed by mistake, now I have added it back again.